src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -191,6 +193,20 @@ message PendingInstanceRequest {
   int32 count = 3;
 }
 
+message PendingInstance {
+  // The instance type name for the cloud, e.g. t1.micro
+  string instance_type_name = 1;
+  // The node type name, e.g. ray-head-node, matching `available_node_types`
+  // in the autoscaler config. See `ray/autoscaler/ray-schema.json`
+  string ray_node_type_name = 2;
+  // The instance id.
+  string instance_id = 3;
+  // The instance ip address.
+  string ip_address = 4;
+  // Detail information about the instance.
+  string details = 5;
```

Comment:
What does details include?

Reply:
maybe zone/detail state information. This doesn't have to be populated if there's none. OSS autoscaler with more info could choose to populate it through this.